.github/CONTRIBUTING.md

```diff
@@ -482,16 +482,62 @@ in the SQL/updates folder.
 
 ### Mapping Standards
 * Map Merge
-  * You MUST run Map Merge prior to opening your PR when updating existing maps to minimize the change differences (even when using third party mapping programs such as FastDMM.)
+  * The following guideline for map merging applies to people who are **NOT** using StrongDMM, please see the StrongDMM section if you are.
+  * You **MUST** run Map Merge prior to opening your PR when updating existing maps to minimize the change differences (even when using third party mapping programs such as FastDMM.)
     * Failure to run Map Merge on a map after using third party mapping programs (such as FastDMM) greatly increases the risk of the map's key dictionary
     becoming corrupted by future edits after running map merge. Resolving the corruption issue involves rebuilding the map's key dictionary;
+    
+* StrongDMM
+  * When using StrongDMM, the following options **MUST** be enabled to avoid file bloat. They can be found under `File > Preferences > Save Options` in SDMM.
+    * Sanitize Variables - Removes variables that are declared on the map, but are the same as default. (For example: A standard floor turf that has dir = 2 declared on the map will have that variable deleted as it is redundant.)
+    * Clean Unused Keys - Removes content tile keys that are no longer used on the map, usually leftover keys from deletions or edits.
+    * Map save format: This **MUST** be set to **TGM** if you do not want to run map merger. Enabling this setting means SDMM will automatically map merge, letting you skip manual merging.
```

Comment:
"Map Merge" rather than "map merger", since the other mentions of it above have it that way.
```suggestion
    * Map save format: This **MUST** be set to **TGM** if you do not want to run Map Merge. Enabling this setting means SDMM will automatically map merge, letting you skip manual merging.
```